### PR TITLE
fix: added alternative stopping criteria for the ik

### PR DESCRIPTION
**Pull request by baszalmstra**
There are a bunch of changes because I formatted the file as well so we get better diffs later on.

### CODE REVIEW

File: orocos_kdl/src/chainiksolverpos_nr_jl.cpp [hunk 1]
```diff
@@ -23,100 +23,133 @@
 
 #include "chainiksolverpos_nr_jl.hpp"
 
+#include <iostream>
```
Comment (MatthijsBurgh) [inline orocos_kdl/src/chainiksolverpos_nr_jl.cpp:26]: Why is this needed? The only `std::cout` is commented. So please remove that one.

File: orocos_kdl/src/chainiksolverpos_nr_jl.cpp [hunk 2]
```diff
@@ -23,100 +23,133 @@
...
 #include <limits>
+//#include <frames_io.hpp>
```
Comment (MatthijsBurgh) [inline orocos_kdl/src/chainiksolverpos_nr_jl.cpp:28]: Remove commented code

File: orocos_kdl/src/chainiksolverpos_nr_jl.cpp [hunk 3]
```diff
@@ -23,100 +23,133 @@
...
-        q_min = q_min_in;
-        q_max = q_max_in;
+    if (i != maxiter)
         return (error = E_NOERROR);
-    }
-
-    ChainIkSolverPos_NR_JL::~ChainIkSolverPos_NR_JL()
+    else
     {
+        // std::cout << "delta_twist " << delta_twist << std::endl;
```
Comment (MatthijsBurgh) [inline orocos_kdl/src/chainiksolverpos_nr_jl.cpp:130]: Remove commented code

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
I understand you in formatting the file. But the formatting is a mess in the entire repo. There is no decision on which styling format to use. Therefore only update the formatting on the lines edited.

P.S. If you want to improve code style, don't add commented  code.

**Comment by baszalmstra** [comment]
Hey @MatthijsBurgh ! Im so sorry Im totally targeting the wrong fork. This should target: https://github.com/smart-robotics/orocos_kinematics_dynamics . There is currently a 3 year old "fix" branch there. I actually want to get rid of that and use this upstream project. To do so, I first wanted to upstream this "fix" branch, this could should definitly not land here. ;)

Again, so sorry for wasting your time!

**Comment by MatthijsBurgh** [comment]
@baszalmstra. No problem. I have roots at @smart-robotics, so happy to work with you to resolve any issues.
